articles/connectors/connectors-native-http.md

```diff
@@ -302,6 +302,17 @@ HTTP requests have a [timeout limit](../logic-apps/logic-apps-limits-and-config.
 
 <a name="disable-location-header-check"></a>
 
+### Retry-After  
+The http action respects the retry after with 429 status code if received one .
```

Comment:
My edit includes a blank line between the heading and body text.

```suggestion

The HTTP trigger and action supports [retry policies using the `Retry-After` value (seconds)](logic-apps-exception-handling.md#retry-policies) along with the [`429 - Too many requests` status code, if received](logic-apps/handle-throttling-problems-429-errors.md), for example:
```

Reply:
OK, the team clarified that `Retry-After` is a header value, so editing my suggestion.

Reply:
I am not sure for the http trigger

Reply:
@mbarqawi: OK, we can remove the trigger if we're not sure. I'll update my revision suggestion.